src/scene/procedural.js

```diff
@@ -639,13 +639,19 @@ pc._createConeData = function (baseRadius, peakRadius, height, heightSegments, c
  */
 pc.createCylinder = function (device, opts) {
     // Check the supplied options and provide defaults for unspecified ones
-    var baseRadius = opts && opts.baseRadius !== undefined ? opts.baseRadius : 0.5;
+    // #ifdef DEBUG
+    if (opts && opts.hasOwnProperty('baseRadius') && !opts.hasOwnProperty('radius'))
+      console.warn('DEPRECATED: "baseRadius" in arguments, use "radius" instead');
```

Comment:
Hi @CarlBateman 
Did you revert this commit ?
https://github.com/playcanvas/engine/pull/973/commits/8eb9e80438d4e5bb746311e17c056ac5f87d7d87